tiledb/cloud/sql/tiledb_connection.py

```diff
@@ -0,0 +1,25 @@
+import tiledb.cloud
+from tiledb.cloud.sql.tiledb_cursor import Cursor
+from tiledb.cloud.sql.tiledb_error import NotSupportedError
+
+
+class TileDBConnection:
+    def __init__(self, token):
+        self.token = token
+        tiledb.cloud.login(token=self.token)
```

Comment:
Lets support all the parameters of `tiledb.cloud.login`.

Also I think we should login only for the python process and not write to the home directory. @thetorpedodog what do you think? I'm thinking the user might not expect passing parameters to the database contructor to effect their global login. I think we'll need a new option for the login function.

Reply:
I didn’t see this comment before—my thought was that we could [assume the user was already logged in](https://github.com/TileDB-Inc/TileDB-Cloud-Py/pull/362#discussion_r1116095551). If we want to do this separately, that’s going to get somewhat hairy since we would need to keep track of a separate client object, API functions, etc. This isn’t necessarily a bad idea, but I think would be a good idea for a follow-up to keep the scope of this change smaller.

Reply:
I'm flexible here, I just don't love the login call. If for v1 we want to force the user to call login outside I'm good with that.

Reply:
The login call was a leftover from when this was a separate package. I think the best way is to have the user call login